srtcore/core.h

```diff
@@ -974,7 +981,7 @@ class CUDT
 
     sync::CThread m_RcvTsbPdThread;              // Rcv TsbPD Thread handle
     sync::Condition m_RcvTsbPdCond;              // TSBPD signals if reading is ready. Use together with m_RecvLock
-    bool m_bTsbPdAckWakeup;                      // Signal TsbPd thread on Ack sent
+    sync::atomic<bool> m_bWakeOnRecv;            // Expected to be woken up when received a packet
```

Comment:
```suggestion
    sync::atomic<bool> m_bWakeOnRecv;            // Expected to be woken up when received a readable packet (complete message).
```

Reply:
The "complete message" could be misleading actually because this field is used exclusively in TSBPD.

Reply:
Maybe that is better

```suggestion
    sync::atomic<bool> m_bWakeOnRecv;            // Expected to wake up TSBPD when received a packet
```

Reply:
Somehow the old name provided more hints on who is to be woken up.
```suggestion
    sync::atomic<bool> m_bWakeOnRecv;            // Expected to wake up TSBPD when a read-ready data packet is received.
```

Reply:
The previous one was called "wake on ACK", while this is no longer valid.